git_pw/api.py

```diff
@@ -136,8 +138,8 @@ def _get(url, params=None, stream=False):
         # 'params' (namely a list of tuples) but it doesn't seem possible to
         # indicate this
         rsp = requests.get(
-            url, auth=_get_auth(), headers=_get_headers(), stream=stream,
-            params=params)  # type: ignore
+            url, auth=_get_auth(optional=True), headers=_get_headers(),
+            stream=stream, params=params)  # type: ignore
```

Comment:
Note to self: the only API that currently requires auth for `GET` requests is `/users`, which `git-pw` doesn't use, so this is good